lib/tasks/demo_data_for_development.rake

```diff
@@ -0,0 +1,70 @@
+namespace :data do
+  desc 'Create demo data for our local development'
+  task test: :environment do
+    include FactoryGirl::Syntax::Methods
+
+    def generate_events program, conference
```

Comment:
but do we want all the conference to have the same events? You could change the `start_hour` of the conference, so the schedule look a little bit different.